### DISCUSSION TITLE
easy bypass for anything (version 86 and above)

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#576] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
(you need an internet brower for this to work)

1) open google on your chromebook
2) type this in the search bar: "jobs hiring near me"
3) make money
4) buy your own chromebook
5) dyknow/goguardian/securly/whateverthefuckunblocking is now gone!!!

i hope this works! upvote and share with a friend!!!

**Comment by amelia-bartlett** [comment]
doesn't work, school only taught me that the mitochondria was the powerhouse of the cell, rather than how to pay taxes

now [HMRC](https://en.wikipedia.org/wiki/HM_Revenue_and_Customs) is after me >:(

(/s if it wasn't obvious)

**Reply by Aka-but-nice** [reply]
LMAO.

**Reply by nathblade16** [reply]
that's my issue :(

**Comment by qDread-dev** [comment]
Alternative method:
run a crypto scam

**Reply by siouxxsies** [reply]
W

**Reply by nathblade16** [reply]
but I want to do it 100% without scams

**Reply by ghost** [reply]
w

**Comment by thebossjeff** [comment]
alternative option:
Download tor and go to a darknet directory. Find a credit carding website, buy a database with money made from the crypto scam you should be running, and use the numbers to buy a shit ton of laptops.

**Reply by qDread-dev** [reply]
Yes

**Reply by nathblade16** [reply]
no

**Reply by quayj16** [reply]
Hey I got a legit one of those, got scammed by like 4 people before learning how to do it. Hit up @goonscardz on telegram

**Comment by Hilololsusy** [comment]
alternate method: steal a gaming PC idk why you would ever get a chromebook

**Reply by Kazuma-Itsuchi** [reply]
Ikr. I don't know why chromebooks exist.

**Reply by nathblade16** [reply]
but I don't want to steal stuff

**Reply by Aka-but-nice** [reply]
You're acting as if you'll go to jail for stealing.

**Reply by nathblade16** [reply]
but I don't steal

**Reply by slurpy303-420** [reply]
como se hace la cripto estafa? paso a paso

**Reply by S-PScripts** [reply]
> How is crypto scam made? Step by step

what?

**Comment by quayj16** [comment]
I didn’t ask that question lol
